src/scripts/main.js

```diff
@@ -357,4 +357,41 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+function calcGender(gender) {
+  if (gender === 'm') {
+    return 'Male';
+  } else {
+    return 'Female';
```

Comment:
Consider using an `else if` statement for better readability and to explicitly handle the case where `gender` is neither 'm' nor 'f'. This can help prevent potential bugs if unexpected values are passed to the function.